includes/nfe-functions.php

```diff
@@ -178,6 +180,7 @@ function nfe_status_label( $status ) {
 		'Cancelled'       => __( 'NFe Cancelled', 'woo-nfe' ),
 		'CancelledFailed' => __( 'NFe Cancelling Failed', 'woo-nfe' ),
 		'IssueFailed'     => __( 'NFe Issuing Failed', 'woo-nfe' ),
+		'sentToNFe'     => __( 'sent to the NFE system', 'woo-nfe' ),
```

Comment:
@JoaoVictorLNacional @souzadavi podemos aqui renomear esse status de **sentToNFe** para **Processing**, bem como a descrição de **sent to the NFE system** para **NFe Processing**.